### ISSUE TITLE
does not read all files in folder

### ISSUE THREAD

**Issue by chrnoe** [BoostIO/Boostnote#1540]
I sync the folder with the notes between two computers, and all worked fine for about one week. Now Boostnote doesn't show all notes anymore; there are 17 files in folder, but Boostnote shows only 10 notes. Reboots, rereads does not have any effect. What can I do?

**Comment by chrnoe** [comment]
there was one file ending with 
    *isStarred: false
    isTrashed: false
    alse*

if I delete this one note, all other notes are loaded, if I delete the last line in this note, this note is loaded properly too. but the wrong saving must be happen in Boostnote.

It would be cool if there were a message, if not all notes could be loaded. And maybe Boostnote could ignore the only one note it couldn't load. if i had dozens of notes, i would not notice a few missing ones.

**Comment by pfftdammitchris** [comment]
Do you know if this happens only after syncing between two computers?

**Comment by chrnoe** [comment]
Until now it did not happen again. And maybe it was a syncing error, if this kind of errors can happen there too. I don't know. Now I take a backup before every sync, so I can research the issue, if it happens again. If this is the case, I place a note here.

**Comment by pfftdammitchris** [comment]
Yes please do because I am a little curious as well. I recall this happening to me when I manually dragged .cson notes (that was created from a different boostnote app folder) into the notes folder, and it began bugging. I want to look into this issue and find out how it is syncing so I can try to help find a better solution to make it less buggy

**Comment by chrnoe** [comment]
Now I ran again in this issue ... I was wondering because I was sure to have some notes, but I didn't find it anymore. But, I have this notes really, but the content of the file seems a little bit destroyed, again. The last lines are
```
  ##### 10. DataRow auslesen mit korrekter Formatierung (Dezimal-Punkt statt Komma)
  >`row.Field<decimal>("RingTime").ToString(CultureInfo.InvariantCulture)`
  
  &nbsp;
  ##### 11. Zeiten messen
  ```c#
  Stopwatch sw = new Stopwatch();
  sw.Start();
  // ...
  sw.Stop();
  Console.WriteLine($"Elapsed={watch.Elapsed}");
  \```
  
  &nbsp;
  ##### 12. DEBUG-Modus feststellen
  ```c#
  #if DEBUG
  			IsDebugMode = Keyboard.Modifiers != ModifierKeys.Shift;
  #else
  			IsDebugMode = false;
  #endif
  \```
'''
tags: [
  "CS"
]
isStarred: true
isTrashed: false
linesHighlighted: []
Shift;
  #else
  			IsDebugMode = false;
  #endif
  \```
  
  &nbsp;
  ##### 13. WPF-Anwendung als Single-Instance-Applikation nur einmal starten
  
  siehe [c# - WPF Single Instance Best Practices - Stack Overflow](https://stackoverflow.com/a/23730146/8640356)
  (zwei unterschiedliche GUIDs notwendig!)
  
'''
tags: [
  "CS"
  "WPF"
]
isStarred: true
isTrashed: false
linesHighlighted: []
```
(Three Backslashs before ''' I have inserted to show the lines here in a codeblock). 
Some information seems doubled. If I delete the doubled lines, the note is loading in boostnote.

I'm still syncing the notes from time to time, but it's only simple file copying. I don't think that the file was corrupted during copying.

**Comment by Flexo013** [comment]
Does the issue persist if you restart Boostnote?
